src/main/java/io/fabric8/jenkins/openshiftsync/BaseWatcher.java

```diff
@@ -34,6 +38,12 @@
     protected ScheduledFuture relister;
     protected final String[] namespaces;
     protected ConcurrentHashMap<String, Watch> watches;
+    protected static ConcurrentHashMap<String, List<PodTemplate>> trackedPodTemplates = new ConcurrentHashMap<String, List<PodTemplate>>();
+    protected static ConcurrentHashMap<String, String> podTemplateToApiType = new ConcurrentHashMap<String, String>();
+    protected static final String cmType = "ConfigMap";
+    protected static final String isType = "ImageStream";
+    private final String PT_NAME_CLAIMED = "The event for %s | %s | %s that attempts to add the pod template %s was ignored because a %s previously created a pod template with the same name";
+    private final String PT_NOT_OWNED = "The event for %s | %s | %s that no longer includes the pod template %s was ignored because the type %s was associated with taht pod template";
```

Comment:
typo - `taht` -> `that`
```suggestion
    private final String PT_NOT_OWNED = "The event for %s | %s | %s that no longer includes the pod template %s was ignored because the type %s was associated with that pod template";
```